cypress/integration/test_input_spec.js

```diff
@@ -203,7 +196,6 @@ describe('default IME conversions', () => {
       .trigger('input')
       .should('have.value', 'かnyな')
       .type('a')
-      .should('have.value', 'かnyaな')
```

Comment:
This was originally checking for the value we _didn't_ want to see `かnyaな`, that wasn't converted until `setRange + trigger` was fired. Fortunately this now seems to be superfluous (at least in Chrome). Either way, the line we care about is after `trigger('input')` where the value has been converted. So I'm leaving these tests in to confirm that we get the result we want at the end, *but* I've removed the line that was checking for the intermediate *incorrect* result since that wasn't really helpful anyway (e.g. in situations like now where it doesn't even occur).